wagtailquickcreate/tests/settings.py

```diff
@@ -52,8 +53,6 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
-
-    'wagtail.core.middleware.SiteMiddleware',
```

Comment:
I remember this is no longer needed in Wagtail 2.15

Reply:
I agreed